recipe/meta.yaml

```
@@ -69,8 +77,6 @@ test:
     {% for lib_ident in lib_idents %}
     - test -f $PREFIX/lib/lib{{ lib_ident }}.dylib  # [osx]
     - test -f $PREFIX/lib/lib{{ lib_ident }}.so  # [linux]
-    - if not exist %PREFIX%/Library/lib/lib{{ lib_ident }}.dll.a exit /b 1  # [win]
-    - if not exist %PREFIX%/Library/bin/msys-{{ lib_ident }}-*.dll exit /b 1  # [win]
```

```suggestion
    - if not exist %LIBRARY_LIB%/lib{{ lib_ident }}.a exit /b 1  # [win]
```
Although, again, should we be converting these into `.lib` files? Potentially producing `.dll`s as well?